codespell_lib/data/dictionary.txt

```diff
@@ -2344,7 +2344,7 @@ inappropiate->inappropriate
 inaugures->inaugurates
 inbalance->imbalance
 inbalanced->imbalanced
-inbetween->between
+inbetween->in between
```

Comment:
My vote would be to recommend "between" (by listing it first) but allow "in between" as second option